### DISCUSSION TITLE
Help with setting up a Waveshare RS485 to RJ45 on Solax X1 G4

### DISCUSSION THREAD

**Discussion by gjpgjp** [wills106/homeassistant-solax-modbus#376] [General]
I've hit a roadblock setting up my Waveshare RS485 to RJ45 Ethernet Converter Module to my Solax X1 G4.

I have the device wired, and it's getting power from inverted, and I have connected the Waveshare to my network via ethernet.  I'm using Vircom to set up DHCP and other bits as described [in the installation notes.](https://github.com/wills106/homeassistant-solax-modbus/wiki/Installation-Notes)

I can't seem to get Vircom to find the device at all, so I'm unable to set up DHCP.  The installation notes say to `use the "vircom" tool (Windows) to set the IP address matching your lan (or to let the device use DHCP; it does not use DHCP by default)`

I would be grateful for some assistance. I don't know what else to try.

**Comment by darkrain-nl** [comment]
The vircom tool can find the device when connected on the same subnet. So the best way to find it is connect the device on the same switch as the pc or laptop you use to configure it and make sure the ports are not in a separated VLAN. My advise is to not use WiFI.

**Reply by gjpgjp** [reply]
Thanks for this, I'm in!
